README.md

```diff
@@ -1,6 +1,25 @@
-# Guiguts-2.0-tkinter
+# Guiguts
+
+Guiguts - application to support creation of books for PG
+
+## Licensing
+
+Copyright Contributors to the Guiguts project at pgdp.org
```

Comment:
```suggestion
Copyright Contributors to the Guiguts project at pgdp.net
```
Although referencing `pgdp.net` might imply the `dproofreaders` project. Maybe:
```
Copyright Contributors to the [Guiguts-py](https://github.com/DistributedProofreaders/guiguts-py) project
```

Reply:
Oops :)